*** src/configure.orig	2015-09-12 14:44:56.813974110 +0000
--- src/configure	2015-09-12 14:44:27.686084528 +0000
*************** ac_hostname=`(hostname || uname -n) 2>/d
*** 568,574 ****
  #
  # Initializations.
  #
! ac_default_prefix=/usr/local
  ac_clean_files=
  ac_config_libobj_dir=.
  LIBOBJS=
--- 568,574 ----
  #
  # Initializations.
  #
! ac_default_prefix=/usr
  ac_clean_files=
  ac_config_libobj_dir=.
  LIBOBJS=
