*** Makefile.in	2013-04-19 04:12:59.000000000 +0200
--- Makefile-patched.in	2015-07-28 09:21:58.000000000 +0200
*************** example-code:
*** 26,44 ****
  	make -C examples
  
  mpsse.o: support.o
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DLIBFTDI1=$(LIBFTDI1) -c mpsse.c
  
  fast.o: support.o
  	$(CC) $(CFLAGS) $(LDFLAGS) -c fast.c
  
  support.o:
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DLIBFTDI1=$(LIBFTDI1) -c support.c
  
  pyswig-build:
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DSWIGPYTHON -DLIBFTDI1=$(LIBFTDI1) -c support.c
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DSWIGPYTHON -DLIBFTDI1=$(LIBFTDI1) -c mpsse.c
  	$(SWIG) -python $(TARGET).i
! 	$(CC) $(CFLAGS) -c  -DLIBFTDI1=$(LIBFTDI1) $(TARGET)_wrap.c  $(INC)
  	$(CC) $(CFLAGS) -shared $(FRAMEWORK) $(TARGET)_wrap.o mpsse.o support.o \
  		-o _pylib$(TARGET).so $(LDFLAGS) $(INC)
  
--- 26,44 ----
  	make -C examples
  
  mpsse.o: support.o
! 	$(CC) $(CFLAGS) $(LDFLAGS) -c mpsse.c
  
  fast.o: support.o
  	$(CC) $(CFLAGS) $(LDFLAGS) -c fast.c
  
  support.o:
! 	$(CC) $(CFLAGS) $(LDFLAGS) -c support.c
  
  pyswig-build:
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DSWIGPYTHON -c support.c
! 	$(CC) $(CFLAGS) $(LDFLAGS) -DSWIGPYTHON -c mpsse.c
  	$(SWIG) -python $(TARGET).i
! 	$(CC) $(CFLAGS) -c  $(TARGET)_wrap.c  $(INC)
  	$(CC) $(CFLAGS) -shared $(FRAMEWORK) $(TARGET)_wrap.o mpsse.o support.o \
  		-o _pylib$(TARGET).so $(LDFLAGS) $(INC)
  
